prodactivity/managers/RoutineManager.tsx

```diff
@@ -33,8 +33,7 @@ export default class RoutineManager {
      */
     async getRoutine(routineId: string): Promise<RoutineViewModel | undefined> {
         try {
-            let response = await fetch(`${this.baseURL}/byroutineid/?routineId=${routineId}`);
-            let routine = await response.json();
+            const routine = await FetchHelper.get(`${this.baseURL}/byroutineid/`, { routineId });
```

Comment:
Just a shower thought, but how many routes are we expecting to have in this application? If we are expecting more than like 5 it would be a good idea to have a route object to easily throw around our application to use everywhere.

Reply:
What do you mean by Route object? Like storing the endpoints? If so, I honestly don't know if it would be beneficial, the components will only ever call the functions defined in the manager classes, so this is the only place an endpoint will ever be implemented.